ClimbingGearBackend/Infrastructure/EFUserGearRepository.cs

```diff
@@ -0,0 +1,59 @@
+using System.Collections.Generic;
+using System.Security.Claims;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.AspNetCore.Identity;
+using ClimbingGearBackend.Models;
+using ClimbingGearBackend.Interfaces;
+
+namespace ClimbingGearBackend.Infrastucture
+{
+  public class EFUserGearRepository : IUserGearRepository
```

Comment:
What is `EF`?

Reply:
Entity Framework, the Microsoft ORM that I'm using.